.eslintrc.js

```diff
@@ -15,6 +15,9 @@
  */
 
 const fs = require('fs');
+const {
+  eslintReportForbiddenTerms,
+} = require('./build-system/tasks/presubmit-checks.js');
```

Comment:
It's weird to import from a task file in `.eslintrc.js`. Would it be possible to do the following?

- Move out all the configuration / allowlists / messages, etc. for `forbiddenTerms` for JS files from `presubmit-checks.js` to a new file called `build-system/test-configs/forbidde-terms.js`
- Have the `local/forbidden-terms` lint rule automatically load its rules from that file

Then, this `require` statement can go away. Also, `amp presubmit` will no longer have anything to do with JS files. I can go in later and figure out what to do with the rest of the `presubmit-checks.js`.

Reply:
Cool, done. I left `requiredTerms` in `presubmit-checks.js` since it's the only user.